aviatrix/resource_aviatrix_spoke_transit_attachment.go

```diff
@@ -130,10 +190,88 @@ func resourceAviatrixSpokeTransitAttachmentRead(d *schema.ResourceData, meta int
 		}
 	}
 
+	if attachment.SpokeBgpEnabled {
+		transitGatewayPeering := &goaviatrix.TransitGatewayPeering{
+			TransitGatewayName1: spokeGwName,
+			TransitGatewayName2: transitGwName,
+		}
+
+		transitGatewayPeering, err = client.GetTransitGatewayPeeringDetails(transitGatewayPeering)
+		if err == goaviatrix.ErrNotFound {
+			d.SetId("")
+			return nil
+		}
+
+		if transitGatewayPeering.PrependAsPath1 != "" {
+			var prependAsPath []string
+			for _, str := range strings.Split(transitGatewayPeering.PrependAsPath1, " ") {
+				prependAsPath = append(prependAsPath, strings.TrimSpace(str))
+			}
+
+			err = d.Set("spoke_prepend_as_path", prependAsPath)
+			if err != nil {
+				return fmt.Errorf("could not set spoke_prepend_as_path: %v", err)
+			}
+		}
+
+		if transitGatewayPeering.PrependAsPath2 != "" {
+			var prependAsPath []string
+			for _, str := range strings.Split(transitGatewayPeering.PrependAsPath2, " ") {
+				prependAsPath = append(prependAsPath, strings.TrimSpace(str))
+			}
+
+			err = d.Set("transit_prepend_as_path", prependAsPath)
+			if err != nil {
+				return fmt.Errorf("could not set transit_prepend_as_path: %v", err)
+			}
+		}
+	}
```

Comment:
We should explicitly set `spoke_prepend_as_path` and `transit_prepend_as_path` to nil if `SpokeBgpEnabled = false`